styles/variables-silicon.less

```diff
@@ -216,6 +216,20 @@
 
 // ToggleButton
 // ---------------------------------------
-@agate-togglebutton-indicator-position: auto 33px 9px 33px;
-@agate-togglebutton-indicator-height: 4px;
-@agate-togglebutton-indicator-border-width: 1px;
+@agate-togglebutton-min-width: 180px;
+@agate-togglebutton-client-padding-left: 24px;
+@agate-togglebutton-huge-min-width: 360px;
+@agate-togglebutton-huge-client-padding-left: 48px;
+@agate-togglebutton-small-min-width: 120px;
+@agate-togglebutton-small-client-padding-left: @agate-togglebutton-client-padding-left;
+@agate-togglebutton-smallest-min-width: 90px;
+@agate-togglebutton-smallest-client-padding-left: @agate-togglebutton-client-padding-left;
+@agate-togglebutton-indicator-border-radius: 50%;
+@agate-togglebutton-indicator-size: 30px;
+@agate-togglebutton-indicator-position: 50% auto auto 30px;
+@agate-togglebutton-indicator-height: 15px;
+@agate-togglebutton-indicator-transform: translateY(-50%);
+@agate-togglebutton-indicator-width: @agate-togglebutton-indicator-height;
+@agate-togglebutton-huge-indicator-position: 50% auto auto 60px;
```

Comment:
If the transform included X, you could simplify the positioning a lot.
```css
@agate-togglebutton-indicator-transform: translate(-50%, -50%);
...
@agate-togglebutton-indicator-position: 50% auto auto 50%
@agate-togglebutton-huge-indicator-position: @agate-togglebutton-indicator-position;
@agate-togglebutton-small-indicator-position: @agate-togglebutton-indicator-position;
@agate-togglebutton-smallest-indicator-position: @agate-togglebutton-indicator-position;
```
Then you wouldn't even need the `huge`, `small`, and `smallest` position variables or their accompanying CSS either, since they'd just all be the same.

Reply:
thanks for the tip.

Reply:
i solved the issue by leaving auto value for position-left
@agate-togglebutton-indicator-position: 50% auto auto;